### PR TITLE
Added skeleton for fdc3 DesktopAgent typescript implementation

### PR DESCRIPTION
**Pull request by ZKRobi**

### CODE REVIEW

File: src/shell/js/composeui-fdc3/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@morgan-stanley/composeui-fdc3",
+  "version": "0.1.0",
+  "description": "FDC3 DesktopAgent implementation for Compose UI",
```
Comment (ztanczos) [inline src/shell/js/composeui-fdc3/package.json:4]: Nitpick: we usually have "ComposeUI" as one word

**Review by ztanczos — DISMISSED** [review]

File: src/shell/js/composeui-fdc3/src/ComposeDesktopAgent.ts [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ *  or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ *  
+ */
+
+
+import { AppIdentifier, AppIntent, AppMetadata, Channel, Context, ContextHandler, DesktopAgent, ImplementationMetadata, IntentHandler, IntentResolution, Listener, PrivateChannel } from '@finos/fdc3'
+
+export class ComposeDesktopAgent implements DesktopAgent {
```
Comment (BalassaMarton) [inline src/shell/js/composeui-fdc3/src/ComposeDesktopAgent.ts:17]: Rename to `ComposeUIDesktopAgent`?

**Review by kruplm — CHANGES_REQUESTED** [review]
The skeleton looks good, but it fails on CI.

Make sure you're adding the package to the lerna monorepo, and also note the error message 

`npm ERR! `npm ci` can only install packages when your package.json and package-lock.json or npm-shrinkwrap.json are in sync. Please update your lock file with `npm install` before continuing.
npm ERR! 
npm ERR! Missing: @morgan-stanley/composeui-fdc3@0.1.0 from lock file
npm ERR! Missing: @morgan-stanley/composeui-fdc3@0.1.0 from lock file
npm ERR! Missing: @finos/fdc3@2.0.1 from lock file
npm ERR! Missing: rimraf@4.1.2 from lock file`

File: src/shell/js/composeui-fdc3/package.json (hunk 1 above)
Comment (ZKRobi) [reply src/shell/js/composeui-fdc3/package.json:4]: I took this form from some other place, it's pretty inconsistent all around - something we should fix :)

**Pull request closed without merging**